RecoTracker/MkFit/plugins/MkFitGeometryESProducer.cc

```diff
@@ -346,6 +399,82 @@ void MkFitGeometryESProducer::addTECGeometry(mkfit::TrackerInfo &trk_info) {
   edm::LogVerbatim("MkFitGeometryESProducer") << ostr.str();
 }
 
+void MkFitGeometryESProducer::findRZBox(const GlobalPoint &gp, float &rmin, float &rmax, float &zmin, float &zmax) {
+  float r = gp.perp(), z = gp.z();
+  if (std::fabs(r) > rmax)
+    rmax = std::fabs(r);
+  if (std::fabs(r) < rmin)
```

Comment:
Could be `else if` if `rmax` is always bigger than `rmin`

Reply:
> use `std::clamp`

`std::min` and `std::max` seem more appropriate clamp would look odd `rmax = clamp(r, rmax, r)`

Reply:
review addressed